boards/sparc/qemu_leon3/Kconfig.board

```diff
@@ -6,3 +6,4 @@ config BOARD_QEMU_LEON3
 	bool "QEMU LEON3 target"
 	depends on SOC_LEON3
 	select QEMU_TARGET
+	select CPU_HAS_FPU
```

Comment:
@martin-aberg I am a little late to the party, but any reason you specified `CPU_HAS_FPU` at board-level instead of SoC-level as done almost everywhere else?

(came across this while reviewing an unrelated PR)